src/python/pants/backend/python/rules/targets.py

```diff
@@ -6,8 +6,6 @@
 
 from pants.backend.python.subsystems.pytest import PyTest
 from pants.build_graph.address import Address
-from pants.engine.fs import Snapshot
```

Comment:
Most changes in this file only move around code. The only real change is to `PythonBinarySources` near the bottom of the file.